site/app/templates/grading/electronic/RubricPanel.twig

```diff
@@ -27,9 +27,18 @@
                     {% if show_verify_all %}
                         <div class="col-no-gutters">
                             <label for="verify-all" tabIndex="0" class="screen-reader">Verify All</label>
-                            <input id="verify-all" type="button" class="btn btn-default key_to_click" tabindex="0" value="Verify All" onclick="onVerifyAll(this)"/>
+                            <input id="verify-all" type="button" class="btn btn-default key_to_click mx-1" tabindex="0" value="Verify All" onclick="onVerifyAll(this)"/>
                         </div>
                     {% endif %}
+
+                    {# clear version conflicts button #}
+                    {% if show_clear_conflicts and not version_conflict %}
+                        <div class="col-no-gutters">
+                            <label for="change-graded-version" tabIndex="0" class="screen-reader">Clear Version Conflicts</label>
+                            <input id="change-graded-version" data-testid="change-graded-version" type="button" class="btn btn-default key_to_click mx-1" tabindex="0" value="Clear Version Conflicts" onclick="updateAllComponentVersions()"/>
```

Comment:
This line is pretty long.  Would it be better to put each attribute on a separate line?

Reply:
Addressed in 87ed1b98d9eb3e92a3bc44dd336c61bb7de3357c